### PR TITLE
oq 1.2.0

### PR DESCRIPTION
**Pull request by Blacksmoke16**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/oq.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
   homepage "https://blacksmoke16.github.io/oq"
-  url "https://github.com/Blacksmoke16/oq/archive/v1.1.2.tar.gz"
-  sha256 "1bd940a72af556a4e685086ca0d3a363d71e3cfedeffb36f865f38d44386f94a"
+  url "https://github.com/Blacksmoke16/oq/archive/v1.2.0.tar.gz"
+  sha256 "15ce979f51045370624e5e50b38ac2eb106a00d5ce9ebf9f04c7fda4a5679826"
   license "MIT"
-  revision 1
 
   bottle do
     rebuild 1
```
Comment (Blacksmoke16) [inline Formula/oq.rb:9]: I imagine this should be removed as well?

Comment (carlocab) [reply Formula/oq.rb]: No; please don't touch the `bottle` block.

**Comment by carlocab** [comment]
You need to add a `depends_on "bdw-gc"`. Squash your commits together when you're done, please. Thanks!

**Comment by Blacksmoke16** [comment]
@carlocab Done.  Thanks!

**Review by carlocab — APPROVED** [review]
Thanks, @Blacksmoke16.

**Pull request closed without merging**